From bb11d1a4ae77d93ec0743e54077cf0f990243fa6 Mon Sep 17 00:00:00 2001
From: Ross Burton <ross.burton@arm.com>
Date: Thu, 28 Mar 2024 16:27:09 +0000
Subject: [PATCH] Skip /timeout/rounding test

This test is sensitive to load because it expects certain timeout operations
to succeed in specific time periods. Whilst these timeouts are fairly large,
they're still exceeded inside a qemu on a loaded system.

https://bugzilla.yoctoproject.org/show_bug.cgi?id=14464

Upstream-Status: Inappropriate [OE-specific]
Signed-off-by: Ross Burton <ross.burton@arm.com>
---
 glib/tests/timeout.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/glib/tests/timeout.c b/glib/tests/timeout.c
index 1ae3f3a34..85a715b0f 100644
--- a/glib/tests/timeout.c
+++ b/glib/tests/timeout.c
@@ -214,7 +214,6 @@ main (int argc, char *argv[])
   g_test_add_func ("/timeout/seconds-once", test_seconds_once);
   g_test_add_func ("/timeout/weeks-overflow", test_weeks_overflow);
   g_test_add_func ("/timeout/far-future-ready-time", test_far_future_ready_time);
-  g_test_add_func ("/timeout/rounding", test_rounding);
 
   return g_test_run ();
 }
-- 
2.34.1

